lib/rubocop/cop/style/multiline_memoization.rb

```diff
@@ -71,6 +86,15 @@ def bad_rhs?(rhs)
             rhs.kwbegin_type?
           end
         end
+
+        def newline_after?(node, pos)
+          node.source_range.source_buffer.source[pos] == "\n"
```

Comment:
We never call `autocorrect` methods in other cops, as far as I know. I think that would be messy.

If it's not too difficult, do the lining up here. If it's overly complicated, just leave it and another cop will step in during the full run.

I don't remember if @bbatsov has stated a clear preference, if it's okay for autocorrect to leave the code a bit untidy and rely on other cops.